src/lib/db/feature-environment-store.ts

```diff
@@ -258,17 +264,22 @@ export class FeatureEnvironmentStore implements IFeatureEnvironmentStore {
     async connectFeatureToEnvironmentsForProject(
         featureName: string,
         projectId: string,
+        config?: [{ environment: string; enabled: boolean }],
```

Comment:
This adds support to pass the desired state of the feature in that environment, which is required for this assertion: https://github.com/Unleash/unleash/pull/2447/files#diff-696f6242d5d467fdce764b004ca9dda90678e55f5051153dd5196d21a9c49e03R463